packages/core/src/browser/common-frontend-contribution.ts

```diff
@@ -1355,6 +1355,61 @@ export class CommonFrontendContribution implements FrontendApplicationContributi
                     hc: '#707070'
                 },
                 description: 'ANSI white background color'
+            },
+
+            // Theia defaults
+            // Buttons
+            {
+                id: 'button.secondary.foreground',
+                defaults: {
+                    dark: Color.white,
```

Comment:
Is it really good default for all themes? the same for `button.secondary.disabled.foreground`